src/content/apis/currency/exchangeratesapi.md

```diff
@@ -0,0 +1,10 @@
+---
+name: exchangeratesapi
+description: The most complete and straightforward JSON API for your exchange rate data needs.
+categories:
+  - currency
+features:
+  https: true
+url: https://exchangeratesapi.io/
+logo: exchangeratesapi_logo_white.png
```

Comment:
We can have the logo to be this instead (without the background)?
![image](https://exchangeratesapi.io/assets/images/exchangeratesapi_logo_white.svg)
https://exchangeratesapi.io/assets/images/exchangeratesapi_logo_white.svg

Reply:
This doesn't seem to be fixed yet, please use the svg image I have attached above 🙏